XCUITest/OnboardingTest.swift

```diff
@@ -10,7 +10,7 @@ class OnboardingTest: XCTestCase {
     override func setUp() {
         super.setUp()
         continueAfterFailure = false
-        app.launchArguments = ["testMode"]
```

Comment:
After "Erase All Contents and Settings" on an iOS simulator, passing in an empty list of `launchArguments` is the only way to get the onboarding screen to show up.